horovod/mxnet/mpi_ops.py

```diff
@@ -95,6 +99,8 @@ def allreduce_(tensor, average=True, name=None):
         average: A flag indicating whether to compute average or summation,
                  defaults to average.
         name: A name of the reduction operation.
+        priority: The priority of this operation. Higher priority operations are
+                  likely to be executed before other actions.
```

Comment:
nit: change the word "actions" to "operations" to be consistent

Reply:
Done